miasm2/os_dep/win_api_x86_32.py

```diff
@@ -1793,13 +1848,39 @@ def msvcrt_memset(jitter):
     jitter.vm.set_mem(args.addr, chr(args.c) * args.size)
     jitter.func_ret_cdecl(ret_ad, args.addr)
 
+def msvcrt_strrchr(jitter):
+    ret_ad, args = jitter.func_args_cdecl(['pstr','c'])
+    s = jitter.get_str_ansi(args.pstr)
+    c = chr(args.c)
+    ret = args.pstr + s.rfind(c)
+    log.info("strrchr(%x '%s','%s') = %x" % (args.pstr,s,c,ret))
+    jitter.func_ret_cdecl(ret_ad, ret)
+
+def msvcrt_wcsrchr(jitter):
+    ret_ad, args = jitter.func_args_cdecl(['pstr','c'])
+    s = jitter.get_str_unic(args.pstr)
+    c = chr(args.c)
+    ret = args.pstr + (s.rfind(c)*2)
+    log.info("wcsrchr(%x '%s',%s) = %x" % (args.pstr,s,c,ret))
+    jitter.func_ret_cdecl(ret_ad, ret)
 
 def msvcrt_memcpy(jitter):
     ret_ad, args = jitter.func_args_cdecl(['dst', 'src', 'size'])
     s = jitter.vm.get_mem(args.src, args.size)
+    #log.info("memcpy buf %s" % s.encode("hex"))
```

Comment:
You still have a commented log here, you can uncomment it

Reply:
Or just remove the line :) ?

Reply:
Your call